docs/30-components/31-table/table-stateful.mdx

```diff
@@ -1,22 +1,21 @@
 ---
-title: Table
-description: Beschreibung, Spezifikation und Beispiele für die Table-Komponente.
+title: Table-Stateful
```

Comment:
```suggestion
title: TableStateful
```

Hier würde ich mich lieber an den bestehenden Seiten orientieren und den Original Komponentennamen in PascalCase-Schreibweise beibehalten.

Ebenfalls in der description und für die andere Seite.